### DISCUSSION TITLE
How do I make each segment 1 word and keep proper timing for youtube shorts style captions

### DISCUSSION THREAD

**Discussion by couchpotatochip21** [openai/whisper#2039] [Q&A]
I have found many other threads detailing things like max_word_count and some other line segment length configurables but no configuration has changed or shortened the word count per segment. I want every word to have it's own segment and be properly timed. I have used the individual word timing configurable and it didn't affect the results.

**Comment by phineas-pta** [comment]
u mean `word_timestamps` ?

**Comment by afsara-ben** [comment]
have you checked the `whisper-timestamped` implementation?
